Readme.md

```diff
@@ -26,6 +26,14 @@ You can install [`lobtui`](https://aur.archlinux.org/packages/lobtui) from the A
 paru -S lobtui
 ```
 
+## 🚩 NetBSD
+
+You cab install [`lobtui`](https://https://pkgsrc.se/editors/lobtui) from the official repositories.
```

Comment:
a small typo here. `can` instead of `cab` :)

Reply:
Sorry ...

Reply:
Fixed now